src/components/DailyTaskList.js

```diff
@@ -0,0 +1,18 @@
+export default function DailyTaskList() {
+  const dailyTask = dailyTasks.map(dailyTask => <li key={dailyTask.toString()}>{dailyTask} </li>);
+  return <ul>{dailyTask}</ul>;
+}
+
+const dailyTasks = [
+  'Räume lüften',
+  'Betten machen',
+  'Spülmaschine ausräumen',
+  'Wäsche machen',
+  'Müll rausbringen',
+  'Böden saugen',
+  'ungenutzte Dinge wegräumen',
+  'Waschbecken im Bad reinigen',
+  'Toilette im Bad reinigen',
+  'Arbeitsflächen in der Küche abwischen',
+  'volle Spülmaschine anschalten',
+];
```

Comment:
I guess you will soon need these information a little bit more above in the component tree; this is why I suggest to save this const in the parent component and pass it as a prop. By doing so, you are creating a "dump" component, which is always a good thing (and you can test it better).